### ISSUE TITLE
adjusting rounding in getElementPosition

### ISSUE THREAD

**Issue by thenewguy** [davidjbradshaw/iframe-resizer#247]
Moved from https://github.com/davidjbradshaw/iframe-resizer/issues/243#issuecomment-136925633

I am observing an odd issue that I couldn't reproduce it at first... if you open the page at 100% zoom it works as expected.  If you change the zoom to 110% it alternates between scrolling to y coord of 429px and 430px.  This was observed at a window width between 548px and 637px according to the chrome web developer plugin.  Is the use of parseInt in [getElementPosition](https://github.com/davidjbradshaw/iframe-resizer/blob/dev/src/iframeResizer.js#L252) important?

Changing it to the following prevents the pixel jumping I am observing:
```
		function getElementPosition(target){
			var iFramePosition = target.getBoundingClientRect();

			getPagePosition(iframeId);

			return {
				x: Math.floor( Number(iFramePosition.left) + Number(pagePosition.x) ),
				y: Math.floor( Number(iFramePosition.top)  + Number(pagePosition.y) )
			};
		}
```

The modified `getElementPosition` function is demonstrated here: http://thenewguy.github.io/iframe-resizer/issues/243/6/host.html

**Comment by davidjbradshaw** [comment]
Do you want to put that in a pull request?

**Comment by thenewguy** [comment]
I am doing that now

**Comment by thenewguy** [comment]
Not sure how important identical whitespace is to you... eclipse is being difficult with it today

**Comment by davidjbradshaw** [comment]
Tabs rather than spaces.

**Comment by davidjbradshaw** [comment]
Merged into dev and will be released in v3.2.0
